node/router/metrics.go

```diff
@@ -109,7 +111,7 @@ func (m *RouterMetrics) trackMetrics() {
 			return
 		case <-ticker.C:
 			txCount := monitoring.GetMetricValue(m.incomingTxs.(prometheus.Metric), m.logger)
-			m.logger.Infof("Received %.f transactions per second", float64(txCount-incomingTxsLastValue)/m.interval.Seconds())
+			m.logger.Infof("ROUTER_METRICS: party_id=%d, received: transactions %d, %.f transactions per second", m.partyID, int(txCount), float64(txCount-incomingTxsLastValue)/m.interval.Seconds())
```

Comment:
`ROUTER_METRICS party_id=%d, transactions_received_total=%d, transactions_received_per_second=%.f`

Reply:
more like the other metrics we already have